pos_receipt_xml_header_footer/readme/USAGE.rst

```diff
@@ -0,0 +1,18 @@
+To use XML (Odoo QWeb) and HTML in the headers and footers of your receipts, do
+the following:
+
+1. Open the Point of Sale application.
+2. Go to Configuration → Point of Sale, and select your POS.
+3. Enable 'Header & Footer'.
+4. Prepend your header or footer with a single line containing ``<!DOCTYPE QWEB``.
+
+For example, to use line breaks::
+
+  <!DOCTYPE QWEB
```

Comment:
shouldn’t this tag be closed? is this an odoo quirk?

Reply:
Odoo quirk. My editor automatically closing the tag caused me at least 30 minutes of frustration :(